app/Http/routes.php

```diff
@@ -37,6 +37,8 @@
 
     Route::resource('cookbooks', 'CookbooksController', ['only' =>
         ['create', 'edit', 'store', 'update', 'destroy']]);
+    Route::post('recipes/{recipe}/postComment', ['as'=>'recipes.postComment', 'uses'=>'RecipesController@postComment']);
+    Route::get('recipes/{recipe}/deleteComment/{comment_id}', ['as'=>'recipes.deleteComment', 'uses'=>'RecipesController@deleteComment']);
```

Comment:
I would prefer the routes to be more RESTful:

```php
Route::post('recipes/{recipe}/comments', ['as'=>'recipes.postComment', 'uses'=>'RecipesController@postComment']);
Route::delete('recipes/{recipe}/comments/{comment_id}', ['as'=>'recipes.deleteComment', 'uses'=>'RecipesController@deleteComment']);
```

For it to work, the HTML forms need to include a `<input type="hidden" name="_method" value="DELETE" />` field.